actions-services.yml

```diff
@@ -0,0 +1,99 @@
+version: '2'
+services:
+    testdb:
+        image: mariadb:10
+        environment:
+            MYSQL_ROOT_PASSWORD: r00tp@ss!
+            MYSQL_DATABASE: test
+            MYSQL_USER: idbroker
+            MYSQL_PASSWORD: idbroker
+
+    app:
+        build: .
+        depends_on:
+            - testdb
+            - mfaapi
+            - u2fsim
+        working_dir: /data
+        environment:
+            APP_ENV: test
+            EMAIL_SERVICE_accessToken: fake-abc-123
+            EMAIL_SERVICE_assertValidIp: "false"
+            EMAIL_SERVICE_baseUrl: http://email
+            EMAIL_SERVICE_validIpRanges: 192.168.0.0/16
+            EMAIL_SIGNATURE: Dummy Signature for Automated Tests
+            EMAILER_CLASS: \Sil\SilIdBroker\Behat\Context\fakes\FakeEmailer
+            HELP_CENTER_URL: https://help-center
+            IDP_NAME: Test
+            MYSQL_HOST: testdb
+            MYSQL_DATABASE: test
+            MYSQL_USER: idbroker
+            MYSQL_PASSWORD: idbroker
```

Comment:
even if it is test, shouldn't these sorts of things be hidden somehow?

Reply:
Oh, it was even in the codeship file. Isn't that still bad?

Reply:
What about this data gives you concern?